src/aiida/cmdline/commands/cmd_computer.py

```diff
@@ -730,3 +730,62 @@ def computer_config_show(computer, user, defaults, as_option_string):
             else:
                 table.append((f'* {name}', '-'))
         echo_tabulate(table, tablefmt='plain')
+
+
+@verdi_computer.group('export', cls=LazyConfigureGroup)
+def computer_export():
+    """Exports the Authinfo details for a computer (and user)."""
+
+
+@computer_export.command('setup')
+@arguments.COMPUTER()
+@arguments.OUTPUT_FILE(type=click.Path(exists=False))
+@with_dbenv()
+def computer_export_setup(computer, output_file):
```

Comment:
As we recently added the option to make the sorting optional for the code export (see PR #6345), I propose we also add it here as an additional option.

Reply:
Personally, I find `append_text` appearing in the beginning of the YAML file confusing, as that is one (if not the) last option one configures when running the command interactively. @sphuber and I discussed this also in the linked PR #6345. While I don't think it's super important, the `aiida-code-registry` and the `aiida-resource-registry` currently contain YAML files which are not alphabetically sorted, but instead in the order in which the options are defined in the `src` which should also be the order in which the user is prompted when running interactively. So if we consider these three use cases: 1) interactive CLI command, 2) import from the YAML files in the registry, and 3) export from a profile, it should be consistent throughout, rather than sorting alphabetically for 3) if that is not done for 1) and 2).